README.md

```diff
@@ -21,11 +23,58 @@ There is an included exe in the releases and /dist folder which SHOULD run, and
 - ![image](https://github.com/user-attachments/assets/719fea74-2d6c-494f-bced-67ebbcc9fbe1)
 
 
-# Python
+### Python
 
 Included is the python script, and a build script if you wish to build it into an .EXE (though ironically, this requires Python if you wish to build - so it is up to you really)
 - **I highly recommend the exe as it is user friendly**
 
 - - You can try building it as stated above and I have included instructions in BUILD_README_FIRST via README_buildexe.bat
 
 ![image](https://github.com/user-attachments/assets/9edc9c69-a67e-46b2-a6dc-0c0457cda087)
+
+## Linux
+
+### .exe
```

Comment:
Oof, this looks as if it would be a lot of work, but it's been really quick to set up.

... Maybe it should be a little less detailed? But on Steam and on itch.io I have lately seen people even ask for installation instructions like "How do I make this file executable?" so my gut feeling rather says, it need be even more detailed.